devices/bosch/room_thermostat2_230V.json

```diff
@@ -0,0 +1,451 @@
+{
+  "schema": "devcap1.schema.json",
+  "manufacturername": "BOSCH",
+  "modelid": "RBSH-RTH0-ZB-EU",
+  "vendor": "Bosch",
+  "product": "Room thermostat II 230V (BTH-RM230Z)",
+  "sleeper": false,
+  "status": "Gold",
+  "subdevices": [
+    {
+      "type": "$TYPE_THERMOSTAT",
+      "restapi": "/sensors",
+      "uuid": [
+        "$address.ext",
+        "0x01",
+        "0x0201"
+      ],
+      "fingerprint": {
+        "profile": "0x0104",
+        "device": "0x0301",
+        "endpoint": "0x01",
+        "in": [
+          "0x0001",
+          "0x0201",
+          "0x0204",
+          "0x0B05"
+        ]
+      },
+      "meta": {
+        "values": {
+          "config/mode": {
+            "auto": 0,
+            "heat": 1,
+            "off": 5
+          }
+        }
+      },
+      "items": [
+        {
+          "name": "attr/id"
+        },
+        {
+          "name": "attr/lastannounced"
+        },
+        {
+          "name": "attr/lastseen"
+        },
+        {
+          "name": "attr/manufacturername"
+        },
+        {
+          "name": "attr/modelid"
+        },
+        {
+          "name": "attr/name"
+        },
+        {
+          "name": "attr/swversion",
+          "refresh.interval": 86400,
+          "parse": {
+            "at": "0x0006",
+            "cl": "0x0000",
+            "ep": 1,
+            "eval": "Item.val = Attr.val",
+            "fn": "zcl:attr"
+          },
+          "read": {
+            "at": "0x0006",
+            "cl": "0x0000",
+            "ep": 1,
+            "fn": "zcl:attr"
+          }
+        },
+        {
+          "name": "attr/type"
+        },
+        {
+          "name": "attr/uniqueid"
+        },
+        {
+          "name": "config/checkin",
+          "public": false,
+          "refresh.interval": 3600,
+          "default": 14400
+        },
```

Comment:
As it has already been mentioned, this part should be removed, as it wouldn't work out based on the provided screenshot.